default/package.json

```
@@ -8,23 +8,23 @@
     "typecheck": "react-router typegen && tsc"
   },
   "dependencies": {
-    "@react-router/node": "*",
-    "@react-router/serve": "*",
-    "isbot": "^5.1.17",
+    "@react-router/node": "^7.2.0",
+    "@react-router/serve": "^7.2.0",
```

Let's keep these as `*`

```suggestion
    "@react-router/node": "*",
    "@react-router/serve": "*",
```